pandas/core/indexes/category.py

```diff
@@ -253,6 +253,10 @@ def get_values(self):
         """ return the underlying data as an ndarray """
         return self._data.get_values()
 
+    def __iter__(self):
+        """ iterate like Categorical """
+        return self._data.__iter__()
```

Comment:
this feels not clean. The `tolist` of Categorical should already ensure this?

Reply:
fixed up